### DISCUSSION TITLE
Support different S3-compatible providers

### DISCUSSION THREAD

**Discussion by savy-91** [fastlane/fastlane#19056] [Ideas]
I was trying to setup Wasabi to store our fastlane match credentials, when I noticed that fastlane match assumes we are using S3 and only allows to set an AWS S3 region.

Most other scripts/cli/tools support configuring a different S3 endpoint and that makes them compatible with Wasabi and Minio (and any other S3 compatible storage provider).

It would be enough to allow the user to set a different endpoint/baseurl for S3.

**Comment by kmahelona** [comment]
We would also appreciate supporting any s3 compatible endpoint rather than just AWS and other Big Tech players. I saw someone made a [PR](https://github.com/fastlane/fastlane/pull/19929) for this but it was closed and the branch was deleted.

**Comment by VladimirTambovtsev** [comment]
I think this is a really missing option.
Why only AWS or google cloud?

Do you need a PR?
